testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -1500,19 +1500,16 @@ fn simple_neon_integration() {
     // query for prometheus metrics
     #[cfg(feature = "monitoring_prom")]
     {
-        wait_for(10, || {
```

Comment:
Pretty sure this `wait_for` was there for a reason. Maybe it's because the thread that runs the metrics hasn't started by the time this function is called?

Reply:
This applies to all the other `wait_for`s below, as well.

Reply:
Yeah, I don't remember my exact thought process, but Jacinta recently had added the `wait_for`s when fixing these in a different PR. I may have just removed them from a bad merge. I've added them back.

It's not that the metrics server hasn't started, but the metric itself isn't updated until somewhere down the relayer stack, and I believe that's why the metric isn't updated "immediately".